src/vidpf.rs

```diff
@@ -45,6 +46,10 @@ pub enum VidpfError {
     #[error("level index out of bounds")]
     IndexLevel,
 
+    /// Error when input's length is larger than the maximum.
+    #[error("invalid label length")]
+    InvalidLabelLength,
```

Comment:
I think I disagree on this one--all of VIDPF is only used in Mastic, and it would mean the VDAF would be passing Mastic errors when it doesn't pass Prio or Poplar errors. Plus the invalid label length is analogous to the invalid weight length and it's a little strange for them to appear at different levels of abstraction.

Reply:
Actually, VIDPF::eval should probably be checking that it doesn't try to evaluate longer inputs than it has correction words.

Reply:
@divergentdave Bumping this

Reply:
Now that we're using `vidpf::Error::InvalidLabelLength` in the VIDPF `eval()` method as well, I'm fine with this.